kuma/wiki/tasks.py

```diff
@@ -56,15 +62,17 @@ def render_stale_documents(log=None):
         # fetch a logger in case none is given
         log = render_stale_documents.get_logger()
 
-    log.info("Found %s stale documents" % stale_docs_count)
+    log.info('Found %s stale documents' % stale_docs_count)
     response = None
     if lock.acquire():
-        try:
-            for doc in stale_docs:
-                doc.render('no-cache', settings.SITE_URL)
-                log.info("Rendered stale %s" % doc)
-        finally:
-            lock.release()
+        stale_pks = stale_docs.values_list('pk', flat=True)
```

Comment:
How big can this get? Would you ever need to chunk it? Can it have multiple pks that might need a `.distinct()`?